plugins/autotools/autotools.php

```diff
@@ -49,7 +49,7 @@ public function set()
 			$this->enable_move = 0;
 			$this->fileop_type = "Move";
 			$this->path_to_finished = "";
-			$this->skip_move_for_files= "/\.rar|\.zip/";
+			$this->skip_move_for_files= "/^[^\/]+(\.rar)|(\.7z)|(\.zip)/";
```

Comment:
False positive (e.g.): precious.and.rare.book.pdf
Regex should check if filename ends by ".7z" , ".rar" (and forgotten variants like .r01,.r02,... and .001,.002,...) or ".zip".
So here is my solution:
`$this->skip_move_for_files = "/(^[^\/]+\.(7z|rar|r\d\d|\d\d\d|zip))$/";`